conf/ocs_version/ocs-4.7.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 DEPLOYMENT:
   default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.7"
```

Comment:
Do we need to change ocs-registry:latest-4.7 to ocs-registry:latest-stable-4.7

Reply:
done